### PR TITLE
Introduce SE version with call Terminating state

### PR DESCRIPTION
**Pull request by deanrobertcook**
For this PR, there should be no noticeable behaviour changes except that for now, calling system messages have been disabled. Will be re-introduced in a future commit to this PR
#### APK
[Download build #11581](http://192.168.120.33:8080/job/Pull%20Request%20Builder/11581/artifact/build/artifact/wire-dev-PR1763-11581.apk)
[Download build #11587](http://192.168.120.33:8080/job/Pull%20Request%20Builder/11587/artifact/build/artifact/wire-dev-PR1763-11587.apk)
[Download build #11589](http://192.168.120.33:8080/job/Pull%20Request%20Builder/11589/artifact/build/artifact/wire-dev-PR1763-11589.apk)
[Download build #11596](http://192.168.120.33:8080/job/Pull%20Request%20Builder/11596/artifact/build/artifact/wire-dev-PR1763-11596.apk)
[Download build #11597](http://192.168.120.33:8080/job/Pull%20Request%20Builder/11597/artifact/build/artifact/wire-dev-PR1763-11597.apk)
[Download build #11609](http://192.168.120.33:8080/job/Pull%20Request%20Builder/11609/artifact/build/artifact/wire-dev-PR1763-11609.apk)
[Download build #11612](http://192.168.120.33:8080/job/Pull%20Request%20Builder/11612/artifact/build/artifact/wire-dev-PR1763-11612.apk)

### CODE REVIEW

**Comment by someoneEsle** [comment]
- [ ] Bug 1
Tapping "end call" in the notification doesn't end the call

**Comment by someoneEsle** [comment]
- [ ] Bug 2
I don't know if it's this PR of if I've messed up this phone completely, but every time I hang up on a group call I have a JOIN button in the conversations list that doesn't go away. Happens with every group conversation I have.

**Comment by someoneEsle** [comment]
- [ ] Bug 3
There is a group call with A, B, C. I am B.
A calls. B and C pick up the phone. Wait for the call to be established.
C hangs up.
B (me) hangs up.
I see a missed call from A in the group chat

File: app/src/main/scala/com/waz/zclient/calling/controllers/CallController.scala [hunk 1]
```diff
@@ -420,6 +429,50 @@ private class ScreenManager(implicit injector: Injector) extends Injectable {
   }
 }
 
+private class GCMManager(callActive: Signal[Boolean])(implicit inject: Injector, ec: EventContext) extends Injectable {
```
Comment (JPonte) [inline app/src/main/scala/com/waz/zclient/calling/controllers/CallController.scala]: GSM

Comment (deanrobertcook) [reply app/src/main/scala/com/waz/zclient/calling/controllers/CallController.scala]: nice one

**Pull request merged**